pxr/usd/usdLux/lightAPI.h

```diff
@@ -435,6 +616,15 @@ class UsdLuxLightAPI : public UsdAPISchemaBase
     /// enableColorTemperature is set to true.  When active, the
     /// computed result multiplies against the color attribute.
     /// See UsdLuxBlackbodyTemperatureAsRgb().
+    /// 
+    /// This is always calculated as an RGB color using a D65 white point,
```

Comment:
The property should have an associated color space, explicitly restricted to one of the canonical color spaces described in the GfColorSpace enumeration. It is then the responsibility of the user to use the facilities in GfColor/ColorSpace to resolve the value into an rgb triple. The Color Interop standard, to which GfColorSpace adheres, has introduced a CIEXY D65 observed canonical space which would be a good choice for the default color space associated with the color temperature.

Reply:
See my comments / questions here: https://github.com/PixarAnimationStudios/OpenUSD/pull/3182#discussion_r1703195632